pkg/login/social/connectors/google_oauth.go

```diff
@@ -24,13 +24,16 @@ const (
 	legacyAPIURL            = "https://www.googleapis.com/oauth2/v1/userinfo"
 	googleIAMGroupsEndpoint = "https://content-cloudidentity.googleapis.com/v1/groups/-/memberships:searchDirectGroups"
 	googleIAMScope          = "https://www.googleapis.com/auth/cloud-identity.groups.readonly"
+	disableHDValidationKey  = "disable_hd_validation"
```

Comment:
proposed change:

`validate_hd`

Reply:
I have updated the configuration option name as requested.